storage/BasicColumnStoreTupleStorageSubBlock.cpp

```diff
@@ -173,45 +173,45 @@ BasicColumnStoreTupleStorageSubBlock::BasicColumnStoreTupleStorageSubBlock(
   }
 }
 
-bool BasicColumnStoreTupleStorageSubBlock::DescriptionIsValid(
+int BasicColumnStoreTupleStorageSubBlock::DescriptionIsValid(
     const CatalogRelationSchema &relation,
     const TupleStorageSubBlockDescription &description) {
   // Make sure description is initialized and specifies BasicColumnStore.
   if (!description.IsInitialized()) {
-    return false;
+    return -1;
```

Comment:
@zuyu Although it's not obvious, changing the return value is preparation for the next PR which completes the BLOCK PROPERTIES feature. The system will create a proto message and then send it to the SubBlockTypeRegistery to get a return code and generate an SQL error explaining the return code. This saves the repeating of all the checks which blocks already do with their validation functions.